# 第一章：微生物组数据中的生物信息分析

本章内容简介：

- 1.1 简单介绍微生物组研究和DNA测序
- 1.2 系统发育分析中的基础概念
- 1.3 16S rRNA测序
- 1.4 鸟枪法宏基因组测序
- 1.5 简要介绍16S rRNA数据分析的两种平台
- 1.6 本章小结

## 1.1 微生物组研究简介

### 1.1.1 什么是人类微生物组？

### 1.1.2 微生物组研究与DNA测序

## 1.2 系统发育分析简介

## 1.3 16S rRNA测序方法

### 1.3.1 16S rRNA测序的优点

### 1.3.2 16S rRNA测序数据的生物信息学分析

#### 1.3.2.1 采样、提取DNA、构建文库

#### 1.3.2.2 DNA测序与质量控制

#### 1.3.2.3 16SrRNA聚类成OTUs

#### 1.3.2.4 16S rRNA测序方法的局限性

## 1.4 鸟枪法宏基因组测序方法

### 1.4.1 宏基因组的定义

### 1.4.2 鸟枪法宏基因组测序的优点

### 1.4.3 鸟枪法宏基因组数据的生物信息学分析

#### 1.4.3.1 采样、提取DNA、构建文库

#### 1.4.3.2 质量控制

#### 1.4.3.3 组装

#### 1.4.3.4 Binning

#### 1.4.3.5 注释

#### 1.4.3.6 鸟枪法宏基因组数据分析的挑战

## 1.5 生物信息数据的分析工具

### 1.5.1 QIIME

### 1.5.2 mothur

### 1.5.3 16S rRNA分析实操

## 1.6 本章小结

## 参考文献

- Acinas, S.G., L.A. Marcelino, et al. 2004. Divergence and redundancy of 16S rRNA sequences in genomes with multiple rrn operons. Journal of Bacteriology 186 (9): 2629–2635.
- Alneberg, J., B.S. Bjarnason, et al. 2014. Binning metagenomic contigs by coverage and composition. Nature Methods 11: 1144.
- Anderson, S. 1981. Shotgun DNA sequencing using cloned DNase I-generated fragments. Nucleic Acids Research 9 (13): 3015–3027.
- Arrieta, M.-C., L.T. Stiemsma, et al. 2014. The intestinal microbiome in early life: Health and disease. Frontiers in Immunology 5: 427.
- Ashelford, K.E., N.A. Chuzhanova, et al. 2005. At least 1 in 20 16S rRNA sequence records currently held in public repositories is estimated to contain substantial anomalies. Applied and Environmental Microbiology 71 (12): 7724–7736.
- Aziz, R.K., D. Bartels, et al. 2008. The RAST server: Rapid annotations using subsystems technology. BMC Genomics 9.
- Bahl, M.I., A. Bergström, et al. 2012. Freezing fecal samples prior to DNA extraction affects the Firmicutes to Bacteroidetes ratio determined by downstream quantitative PCR analysis. FEMS Microbiology Letters 329 (2): 193–197.
- Bibby, K. 2013. Metagenomic identiﬁcation of viral pathogens. Trends in Biotechnology 31 (5): 275–279.
- Bond, P.L. 1995. Bacterial community structures of phosphate-removing and non-phosphate- removing activated sludges from sequencing batch reactors. Applied and Environment Microbiology 61: 1910–1916.
- Borneman, J., and E.W. Triplett. 1997. Molecular microbial diversity in soils from eastern Amazonia: Evidence for unusual microorganisms and microbial population shifts associated with deforestation. Applied and Environmental Microbiology 63 (7): 2647–2653.
- Brown, S.P., A.M. Veach, et al. 2015. Scraping the bottom of the barrel: Are rare high throughput sequences artifacts? Fungal Ecology 13: 221–225.
- Bulgarelli, D., K. Schlaeppi, et al. 2013. Structure and functions of the bacterial microbiota of plants. Annual Review of Plant Biology 64 (1): 807–838.
- Caporaso, J.G., J. Kuczynski, et al. 2010. QIIME allows analysis of high-throughput community sequencing data. Nature Methods 7: 335.
- Chen, K., and L. Pachter. 2005. Bioinformatics for whole-genome shotgun sequencing of microbial communities. PLoS Computational Biology 1 (2): e24.
- Claesson, M.J., Q. Wang, et al. 2010. Comparison of two next-generation sequencing technologies for resolving highly complex microbiota composition using tandem variable 16S rRNA gene regions. Nucleic Acids Research 38 (22): 29.
- Cole, J.R., Q. Wang, et al. 2009. The ribosomal database project: Improved alignments and new tools for rRNA analysis. Nucleic Acids Research 37(Database issue): D141–D145.
- Cole, J.R., Q. Wang, et al. 2014. Ribosomal Database Project: Data and tools for high throughput rRNA analysis. Nucleic Acids Research 42 (D1): D633–D642.
- D’Amore, R., U.Z. Ijaz, et al. 2016. A comprehensive benchmarking study of protocols and sequencing platforms for 16S rRNA community proﬁling. BMC Genomics 17 (55): 015–2194. DeSantis, T.Z., P. Hugenholtz, N. Larsen, M. Rojas, E.L. Brodie, K. Keller, T. Huber, D. Dalevi,
- P. Hu and G.L.Andersen. 2006. Greengenes, a Chimera-Checked 16S rRNA Gene Database and Workbench Compatible withARB. 72 (7): 5069–5072.
- DiGiulio, D.B. 2012. Diversity of microbes in amniotic fluid. Seminars in Fetal and Neonatal Medicine 17 (1): 2–11.
- DiGiulio, D.B., R. Romero, et al. 2008. Microbial prevalence, diversity and abundance in amniotic fluid during preterm labor: A molecular and culture-based investigation. PLoS ONE 3 (8): e3056.
- Eckburg, P.B., E.M. Bik, et al. 2005. Diversity of the human intestinal microbial flora. Science 308 (5728): 1635–1638.
- Edgar, R.C. 2010. Search and clustering orders of magnitude faster than BLAST. Bioinformatics 26.
- Edgar, R. 2016. UCHIME2: Improved chimera prediction for amplicon sequencing. bioRxiv.
- Edgar, R.C., and H. Flyvbjerg. 2015. Error ﬁltering, pair assembly and error correction for next-generation sequencing reads. Bioinformatics 31 (21): 3476–3482.
- Edgar, R.C., B.J. Haas, et al. 2011. UCHIME improves sensitivity and speed of chimera detection. Bioinformatics 27 (16): 2194–2200.
- Ercolini, D. 2017. Exciting strain-level resolution studies of the food microbiome. Microbial Biotechnology 10 (1): 54–56.
- Everett, K.D., R.M. Bush, et al. 1999. Emended description of the order Chlamydiales, proposal of Parachlamydiaceae fam. nov. and Simkaniaceae fam. nov., each containing one monotypic genus, revised taxonomy of the family Chlamydiaceae, including a new genus and ﬁve new species, and standards for the identiﬁcation of organisms. International Journal of Systematic Bacteriology 2: 415–440.
- Everitt, Brian S., Sabine Landau, et al. 2011. Cluster analysis. Chichester: Wiley.
- Gantner, S., A. F. Andersson, L. Alonso-Sáez and S. Bertilsson (2011). Novel primers for 16S rRNA-basedarchaeal community analyses in environmental samples. Journal of Microbiological Methods 84 (1): 12–18.
- Garcia-Garcerà, M., K. Garcia-Etxebarria, et al. 2013. A new method for extracting skin microbes allows metagenomic analysis of whole-deep skin. PLoS ONE 8 (9): e74914.
- Gilbert, J.A., D. Field, et al. 2010. The taxonomic and functional diversity of microbes at a temperate coastal site: A ‘multi-omic’ study of seasonal and diel temporal variation. PLoS ONE 5 (11): 0015545.
- Godzik, A. 2011. Metagenomics and the protein universe. Current Opinion in Structural Biology 21 (3): 398–403.
- Goodrich, Julia K., Sara C. Di Rienzi, et al. 2014. Conducting a microbiome study. Cell 158 (2): 250–262.
- Greenblum, S., R. Carr, et al. 2015. Extensive strain-level copy-number variation across human gut microbiome species. Cell 160 (4): 583–594.
- Handelsman, J. 2001. Metagenomics and microbial communities. New York: Wiley. Handelsman, J. 2004. Metagenomics: Application of genomics to uncultured microorganisms.
- Microbiology and Molecular Biology Reviews 68 (4): 669–685.
- Heather, J.M., and B. Chain. 2016. The sequence of sequencers: The history of sequencing DNA. Genomics 107 (1): 1–8.
- Holley, R.W., J. Apgar, et al. 1965. Structure of a ribonucleic acid. Science 147 (3664): 1462– 1465.
- Hollmer, M. 2013. Roche to close 454 life sciences as it reduces gene sequencing focus. Fierce Biotechnology.
- Hugenholtz, P., B.M. Goebel, et al. 1998. Impact of culture-independent studies on the emerging phylogenetic view of bacterial diversity. Journal of Bacteriology 180 (18): 4765–4774.
- Huson, D.H., A.F. Auch, et al. 2007. MEGAN analysis of metagenomic data. Genome Research 17 (3): 377–386.
- Huson, D.H., A.F. Auch, et al. 2007. Metagenome analysis using MEGAN. In Proceedings of the 5th Asia-Paciﬁc Bioinformatics Conference, Volume 5 of Series on Advances in Bioinformatics and Computational Biology, ed. D. Sankoff, L. Wang, and F. Chin.
- Huson, D.H., D.C. Richter, et al. 2009. Methods for comparative metagenomics. BMC Bioinformatics 10 (1): S12.
- iHMP 2014. The integrative human microbiome project: Dynamic analysis of microbiome-host omics proﬁles during periods of human health and disease. Cell Host Microbe 16(3): 276–289.
- Jimenez, E., L. Fernandez, et al. 2005. Isolation of commensal bacteria from umbilical cord blood of healthy neonates born by cesarean section. Current Microbiology 51 (4): 270–274.
- Jiménez, E., M.L. Marín, et al. 2008. Is meconium from healthy newborns actually sterile? Research in Microbiology 159 (3): 187–193.
- Jünemann, S., K. Prior, et al. 2014. GABenchToB: A genome assembly benchmark tuned on bacteria and benchtop sequencers. PLoS ONE 9 (9): e107014.
- Jünemann, S., N. Kleinbölting, et al. 2017. Bioinformatics for NGS-based metagenomics and the application to biogas research. Journal of Biotechnology 261: 10–23.
- Kembel, S.W., M. Wu, et al. 2012. Incorporating 16S gene copy number information improves estimates of microbial diversity and abundance. PLoS Computational Biology 8 (10): e1002743.
- Kim, M., M. Morrison, et al. 2011. Evaluation of different partial 16S rRNA gene sequence regions for phylogenetic analysis of microbiomes. Journal of Microbiol Methods 84 (1): 81–87.
- Kuczynski, J., C.L. Lauber, et al. 2011. Experimental and analytical tools for studying the human microbiome. Nature Reviews Genetics 13: 47.
- Kuczynski, J., J. Stombaugh, et al. 2012. Using QIIME to analyze 16S rRNA gene sequences from microbial communities. Current Protocols in Microbiology 1: Unit-1E.5.
- Kunin, V., A. Copeland, et al. 2008. A bioinformatician’s guide to metagenomics. Microbiology and Molecular Biology Reviews: MMBR 72 (4): 557–578.
- Lane, D.J., B. Pace, et al. 1985. Rapid determination of 16S ribosomal RNA sequences for phylogenetic analyses. Proceedings of the National Academy of Sciences of the United States of America 82 (20): 6955–6959.
- Lederberg J., and A. McCray. 2001. Ome sweet omics: a genealogical treasury of words. The Scientist 15: 8
- Legendre, P., and L. Legendre. 2012. Numerical ecology. Amsterdam: Elsevier.
- Lewandowska, D.W., O. Zagordi, et al. 2015. Unbiased metagenomic sequencing complements speciﬁc routine diagnostic methods and increases chances to detect rare viral strains. Diagnostic Microbiology and Infectious Disease 83 (2): 133–138.
- Ley, R.E., D.A. Peterson, et al. 2006. Ecological and evolutionary forces shaping microbial diversity in the human intestine. Cell 124 (4): 837–848.
- Liu, L., Y. Li, et al. 2012. Comparison of next-generation sequencing systems. Journal of Biomedicine and Biotechnology 2012: 11.
- Logares, R., S. Sunagawa, et al. 2014. Metagenomic 16S rDNA Illumina tags are a powerful alternative to amplicon sequencing to explore diversity and structure of microbial communities. Environmental Microbiology 16 (9): 2659–2671.
- Mackie, R.I., A. Sghir, et al. 1999. Developmental microbial ecology of the neonatal gastrointestinal tract. American Journal of Clinical Nutrition 69 (5): 1035S–1045S.
- Madan, J.C., R.C. Salari, et al. 2012. Gut microbial colonisation in premature neonates predicts neonatal sepsis. Archives of Disease in Childhood. Fetal and Neonatal Edition 97 (6): F456–F462.
- Margulies, M., M. Egholm, et al. 2005. Genome sequencing in open microfabricated high density picoliter reactors. Nature 437 (7057): 376–380.
- Markowitz, V.M., K. Mavromatis, et al. 2009. IMG ER: A system for microbial genome annotation expert review and curation. Bioinformatics 25 (17): 2271–2278.
- Matamoros, S., C. Gras-Leguen, et al. 2013. Development of intestinal microbiota in infants and its impact on health. Trends in Microbiology 21 (4): 167–173.
- McCaig, A.E., L.A. Glover, et al. 1999. Molecular analysis of bacterial community structure and diversity in unimproved and improved upland grass pastures. Applied and Environment Microbiology 65 (4): 1721–1730.
- McOrist, A.L., M. Jackson, et al. 2002. A comparison of ﬁve methods for extraction of bacterial DNA from human faecal samples. Journal of Microbiological Methods 50 (2): 131–139.
- Metzker, M.L. 2005. Emerging technologies in DNA sequencing. Genome Research 15 (12): 1767–1776.
- Metzker, M.L. 2010. Sequencing technologies—the next generation. Nature Reviews Genetics 11 (1): 31–46.
- Modolo, L., and E. Lerat. 2015. UrQt: An efﬁcient software for the unsupervised quality trimming of NGS data. BMC Bioinformatics 16 (1): 137.
- Morgan, X.C., and C. Huttenhower. 2012. Human microbiome analysis. PLOS Computational Biology 8(12): e1002808.
- National Research Council. 2007. The new science of metagenomics: Revealing the secrets of our microbial planet. Washington, DC: The National Academies Press.
- NIH. 2010. Human microbiome project—core microbiome sampling protocol A. HMP Initiative 1: Core Microbiome Sampling Protocol A Version 9.0, 29 Mar 2010 (HMP Protocol Number: HMP-07-001).
- Nilakanta, H., K.L. Drews, et al. 2014. A review of software for analyzing molecular sequences. BMC Research Notes 7 (1): 830.
- Oulas, A., C. Pavloudi, et al. 2015. Metagenomics: Tools and insights for analyzing next-generation sequencing data derived from biodiversity studies. Bioinformatics and Biology Insights 9: 75–88.
- Ó Cuív, P., C.D. Aguirre de Carcer, et al. 2011. The effects from DNA extraction methods on the evaluation of microbial diversity associated with human colonic tissue. Microbial Ecology 61(2): 353–362.
- Penders, J., C. Thijs, et al. 2006. Factors influencing the composition of the intestinal microbiota in early infancy. Pediatrics 118 (2): 511–521.
- Plummer, E, J. Twin, D.M. Bulach, S.M. Garland, and S.N. Tabrizi. 2015. A comparison of three bioinformatics pipelines for the analysis of preterm Gut Microbiota using 16S rRNA gene sequencing data. Journal of Proteomics & Bioinformatics 8: 283–291.
- Pruesse, E., C. Quast, K. Knittel, B.M. Fuchs, W. Ludwig, J. Peplies and F.O. Glöckner. 2007. SILVA: acomprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible withARB. Nucleic Acids Research 35 (21): 7188–7196.
- Qin, J., R. Li, et al. 2010. A human gut microbial gene catalogue established by metagenomic sequencing. Nature 464: 59.
- Quince, C., A. Lanzen, et al. 2011. Removing noise from pyrosequenced amplicons. BMC Bioinformatics 12 (1): 38.
- Ravin, N.V., A.V. Mardanov, et al. 2015. Metagenomics as a tool for the investigation of uncultured microorganisms. Russian Journal of Genetics 51 (5): 431–439.
- Reeder, J., and R. Knight. 2010. Rapidly denoising pyrosequencing amplicon reads by exploiting rank-abundance distributions. Nature Methods 7(9): 668–669. https://doi.org/10.1038/ nmeth0910-668b.
- Relman, D.A. 2013. Metagenomics, infectious disease diagnostics, and outbreak investigations: Sequence ﬁrst, ask questions later? JAMA 309 (14): 1531–1532.
- Riesenfeld, C.S., P.D. Schloss, et al. 2004. Metagenomics: Genomic analysis of microbial communities. Annual Review of Genetics 38 (1): 525–552.
- Sait, M., P. Hugenholtz, et al. 2002. Cultivation of globally distributed soil bacteria from phylogenetic lineages previously only detected in cultivation-independent surveys. Environmental Microbiology 4 (11): 654–666.
- Sanger, F., G.G. Brownlee, et al. 1965. A two-dimensional fractionation procedure for radioactive nucleotides. Journal of Molecular Biology 13(2): 373–374.
- Sanger, F., and A.R. Coulson. 1975. A rapid method for determining sequences in DNA by primed synthesis with DNA polymerase. Journal of Molecular Biology 94 (3): 441–448.
- Sanger,  F.,  S.  Nicklen,  et  al.  1977.  DNA  sequencing  with  chain-terminating  inhibitors.
- Proceedings of National Academic Science United States of America 74 (12): 5463–5467. Schadt,  E.E.,  S.  Turner,  et  al.  2010.  A  window  into  third-generation  sequencing.  Human Molecular Genetics 19 (R2): 21.
- Schirmer, M., U.Z. Ijaz, et al. 2015. Insight into biases and sequencing errors for amplicon sequencing with the Illumina MiSeq platform. Nucleic Acids Research 43 (6): 13.
- Schloss, P.D., and J. Handelsman. 2004. Status of the microbial census. Microbiology and Molecular Biology Reviews 68 (4): 686–691.
- Schloss, P.D., and J. Handelsman. 2005. Introducing DOTUR, a computer program for deﬁning operational taxonomic units and estimating species richness. Applied and Environment Microbiology 71 (3): 1501–1506.
- Schloss, P.D., and J. Handelsman. 2008. A statistical toolbox for metagenomics: Assessing functional diversity in microbial communities. BMC Bioinformatics 9 (1): 34.
- Schloss, P.D., and S.L. Westcott. 2011. Assessing and improving methods used in operational taxonomic unit-based approaches for 16S rRNA gene sequence analysis. Applied and Environment Microbiology 77 (10): 3219–3226.
- Schloss, P.D., S.L. Westcott, et al. 2009. Introducing mothur: Open-source, platform-independent, community-supported software for describing and comparing microbial communities. Applied and Environment Microbiology 75 (23): 7537–7541.
- Schloss, P.D., D. Gevers, et al. 2011. Reducing the effects of PCR ampliﬁcation and sequencing artifacts on 16S rRNA-based studies. PLoS ONE 6 (12): e27310.
- Schmieder, R., and R. Edwards. 2011. Fast identiﬁcation and removal of sequence contamination from genomic and metagenomic datasets. PLoS ONE 6 (3): e17288.
- Scholz, M.B., C.-C. Lo, et al. 2012. Next generation sequencing and bioinformatic bottlenecks: The current state of metagenomic data analysis. Current Opinion in Biotechnology 23 (1): 9–15.
- Scholz, M., A. Tett, et al. 2015. Computational tools for taxonomic microbiome proﬁling of shotgun metagenomes A2. In Metagenomics for microbiology, ed. I. Jacques and M.C. Rivera, 67–80. Oxford: Academic Press.
- Sczyrba, A., P. Hofmann, et al. 2017. Critical assessment of metagenome interpretation—a benchmark of computational metagenomics software. bioRxiv.
- Segata, N., D. Boernigen, et al. 2013. Computational meta’omics for microbial community studies. Molecular Systems Biology 9(1).
- Sender, R., S. Fuchs and R. Milo. 2016. Revised estimates for the number of human and bacteria cells inthe body. PLOS Biology 14 (8): e1002533.
- Sharpton, T.J. 2014. An introduction to the analysis of shotgun metagenomic data. Frontiers in Plant Science 5(209).
- Sharpton, T.J., S.J. Riesenfeld, et al. 2011. PhylOTU: A high-throughput procedure quantiﬁes microbial community diversity and resolves novel taxa from metagenomic data. PLoS Computational Biology 7 (1): e1001061.
- Sim, K., M.J. Cox, et al. 2012. Improved detection of biﬁdobacteria with optimised 16S rRNA-Gene Based Pyrosequencing. PLoS ONE 7 (3): e32543.
- Stackebrandt, E., and B.M. Goebel. 1994. Taxonomic Note: A Place for DNA-DNA Reassociation and 16 s rRNA Sequence Analysis in the Present Species Deﬁnition in Bacteriology.
- Staden, R. 1979. A strategy of DNA sequencing employing computer programs. Nucleic Acids Research 6 (7): 2601–2610.
- Stearns, J.C., M.D.J. Lynch, et al. 2011. Bacterial biogeography of the human digestive tract.
- Scientiﬁc Reports 1: 170.
- Strous, M., B. Kraft, et al. 2012. The binning of metagenomic contigs for microbial physiology of mixed cultures. Frontiers in Microbiology 3: 410.
- Sun, J., and P.K. Dudeja. 2018. Introduction. In Mechanisms underlying host-microbiome interactions in pathophysiology of human diseases, ed. J. Sun and P.K. Dudeja. New York: Springer.
- Thomas, T., J. Gilbert, et al. 2012. Metagenomics—A guide from sampling to data analysis. Microbial Informatics and Experimentation 2 (1): 3.
- Tyler, A.D., M.I. Smith, et al. 2014. Analyzing the human microbiome: A “How To” guide for physicians. The American Journal of Gastroenterology 109: 983.
- Ursell, L.K., J.L. Metcalf, et al. 2012. Deﬁning the human microbiome. Nutrition Reviews 70 (Suppl 1): S38–S44.
- van Dijk, E.L., H. Auger, et al. 2014a. Ten years of next-generation sequencing technology.Trends in Genetics 30 (9): 418–426.
- van Dijk, E.L., Y. Jaszczyszyn, et al. 2014b. Library preparation methods for next-generation sequencing: Tone down the bias. Experimental Cell Research 322 (1): 12–20.
- Vázquez-Baeza, Y., A. Gonzalez, et al. 2017. Guiding longitudinal sampling in IBD cohorts. Gut. Vorholt, J.A. 2012. Microbial life in the phyllosphere. Nature Reviews Microbiology 10 (12): 828–840.
- Watson, J.D., and F.H. Crick. 1953. Molecular structure of nucleic acids; a structure for deoxyribose nucleic acid. Nature 171 (4356): 737–738.
- Weisburg, W.G., S.M. Barns, et al. 1991. 16S ribosomal DNA ampliﬁcation for phylogenetic study. Journal of Bacteriology 173 (2): 697–703.
- Werner, J. 2014. QIIME Overview Tutorial.
- Whitman, W.B., D.C. Coleman, et al. 1998. Prokaryotes: The unseen majority. Proceedings of the National Academic Science United States of America 95 (12): 6578–6583.
- Wiley, E.O., and B.S. Lieberman. 2011. Phylogenetics: Theory and practice of phylogenetic systematics. Hoboken: Wiley-Blackwell.
- Woese, C.R., and G.E. Fox. 1977. Phylogenetic structure of the prokaryotic domain: The primary kingdoms. Proceedings of the National Academy of Sciences of the United States of America 74 (11): 5088–5090.
- Woese, C.R., L.J. Magrum, et al. 1978. Archaebacteria. Journal of Molecular Evolution 11 (3): 245–251.
- Woese, C.R., O. Kandler, et al. 1990. Towards a natural system of organisms: Proposal for the domains Archaea, Bacteria, and Eucarya. Proceedings of the National Academy of Sciences 87 (12): 4576–4579.
- Wommack, K.E., J. Bhavsar, et al. 2008. Metagenomics: Read length matters. Applied and Environment Microbiology 74 (5): 1453–1463.
- Wrighton, K.C., B.C. Thomas, et al. 2012. Fermentation, hydrogen, and sulfur metabolism in multiple uncultivated bacterial phyla. Science 337 (6102): 1661–1665.
- Wu, G.D., and J.D. Lewis. 2013. Analysis of the human gut microbiome and association with disease. Clinical Gastroenterology and Hepatology 11 (7): 774–777.
- Wu, G.D., F.D. Bushmanc, et al. 2013. Diet, the human gut microbiota, and IBD. Anaerobe 24: 117–120.
- Wu, W.-K., C.-C. Chen, et al. 2018. Optimization of fecal sample processing for microbiome study—the journey from bathroom to bench. Journal of the Formosan Medical Association.
- Wylie, K.M., R.M. Truty, et al. 2012. Novel bacterial taxa in the human microbiome. PLoS ONE 7 (6): e35294.
- Xia, Y., and J. Sun. 2017. Hypothesis testing and statistical analysis of microbiome. Genes & Diseases 4 (3): 138–148.
- Yozwiak, N.L., P. Skewes-Cox, et al. 2012. virus identiﬁcation in unknown tropical febrile illness cases using deep sequencing. PLoS Neglected Tropical Diseases 6 (2): e1485.
- Yuan, S., D.B. Cohen, et al. 2012. Evaluation of methods for the extraction and puriﬁcation of dna from the human microbiome. PLoS ONE 7 (3): e33865.
- Zhang, C., and L. Zhao. 2016. Strain-level dissection of the contribution of the gut microbiome to human metabolic disease. Genome Medicine 8 (1): 016–0304.
- Zhang, C., S. Li, et al. 2013. Structural modulation of gut microbiota in life-long calorie-restricted mice. Nature Communications 4: 2163.
- Zhao, L., F. Zhang, et al. 2018. Gut bacteria selectively promoted by dietary ﬁbers alleviate type 2 diabetes. Science 359 (6380): 1151–1156.
