# 第十章：微生物组数据的组成性分析

- 10.1 Introduction to Compositional Analysis
- 10.2 Why Microbiome Dataset Can Be Treated as Compositional?
- 10.3 Exploratory Compositional Data Analysis
- 10.4 Comparison Between the Groups Using ALDEx2 Package
- 10.5 Proportionality: Correlation Analysis for Relative Data
- 10.6 总结与讨论

## 10.1 Introduction to Compositional Analysis

## 10.2 Why Microbiome Dataset Can Be Treated as Compositional?

## 10.3 Exploratory Compositional Data Analysis

## 10.4 Comparison Between the Groups Using ALDEx2 Package

## 10.5 Proportionality: Correlation Analysis for Relative Data

## 10.6 总结与讨论

## 参考文献

- Aitchison, J. 1981. A new approach to null correlations of proportions. Mathematical Geology 13 (2): 175–189.
- Aitchison, J. 1982. The statistical analysis of compositional data (with discussion). Journal of the Royal Statistical Society, Series B (Statistical Methodology) 44 (2): 139–177.
- Aitchison, J. 1983. Principal component analysis of compositional data. Biometrika 70 (1): 57–65. Aitchison, J. 1984. Reducing the dimensionality of compositional data sets. Journal of the International Association for Mathematical Geology 16 (6): 617–635.
- Aitchison, J. 1986. The statistical analysis of compositional data. London: Chapman and Hall Ltd. Reprinted in 2003 with additional material by The Blackburn Press.
- Aitchison, J. 1999. A concise guide to compositional data analysis. Posted by Vincent Granville on August 5, 2013 at 9:41am in New Books and Journals.
- Aitchison, J. 2003. A concise guide to compositional data analysis. In 2nd Compositional Data Analysis Workshop. Girona, Italy.
- Aitchison, J., and J.A.C. Brown. 1969. The lognormal distribution with special reference to its uses in econometrics. Cambridge, UK: Cambridge University Press.
- Aitchison, J., and J.J. Egozcue. 2005. Compositional data analysis: Where are we and where should we be heading? Mathematical Geology 37 (7): 829–850.
- Aitchison, J., and M. Greenacre. 2002. Biplots of compositional data. Journal of the Royal Statistical Society: Series C (Applied Statistics) 51 (4): 375–392.
- Aitchison, J., and J. Kay. 2003. Possible solution of some essential zero problems in compositional data analysis. In Proceedings of CoDaWork’03, The 1st Compositional Data Analysis Workshop. Girona, Spain: University of Girona. http://ima.ud.es/Activitats/CoDaWork03/.
- Aitchison, J., C. Barceló-Vidal, et al. 2000. Logratio analysis and compositional distance. Mathematical Geology 32 (3): 271–275.
- Altman, D.G., and J.M. Bland. 1983. Measurement in medicine: The analysis of method comparison studies. Journal of the Royal Statistical Society. Series D (The Statistician) 32 (3): 307–317.
- Arumugam, M., J. Raes, et al. 2011. Enterotypes of the human gut microbiome. Nature 473 (7346): 174–180.
- Bacon-Shone, J. 2003. Modelling structural zeros in compositional data. In Proceedings of CoDaWork’03, The 1st Compositional Data Analysis Workshop. Girona, Spain: University of Girona. http://ima.ud.es/Activitats/CoDaWork03/.
- Bacon-Shone, J. 2008. Discrete and continuous compositions. In Proceedings of CODAWORK’08, The 3rd Compositional Data Analysis Workshop, ed. J. Daunis-i Estadella and J. Martín-Fernández. Girona: University of Girona.
- Bacon-Shone, J. 2011. A short history of compositional data analysis. In Compositional data analysis: Theory and applications, ed. V. Pawlowsky-Glahn and A. Buccianti. Chichester, UK: Wiley.
- Barceló-Vidal, C., J.A. Martín-Fernández, et al. 2001. Mathematical foundations of compositional data analysis. In Proceedings of IAMG.
- Barnett, V. 1981. Interpreting multivariate data. New York: Wiley.
- Bland, J.M., and D.G. Altman. 1999. Measuring agreement in method comparison studies. Statistical Methods in Medical Research 8 (2): 135–160.
- Butte, A.J., and I.S. Kohane. 2000. Mutual information relevance networks: Functional genomic clustering using pairwise entropy measurements. In Paciﬁc Symposium on Biocomputing, 418–429.
- Cameron, A.C., and P.K. Trivedi. 1998. Regression analysis of count data. Cambridge, UK: Cambridge University Press.
- Chen, E.Z., and H. Li. 2016. A two-part mixed-effects model for analyzing longitudinal microbiome compositional data. Bioinformatics 32 (17): 2611–2617.
- Eaton, M.L. 1983. Multivariate statistics. A vector space approach, 512. New York: Wiley. Egozcue, J.J. 2009. Reply to “On the Harker variation diagrams; …” by J.A. Cortés. Mathematical Geosciences 41 (7): 829–834.
- Egozcue, J.J., and V. Pawlowsky-Glahn. 2005. Groups of parts and their balances in compositional data analysis. Mathematical Geology 37 (7): 795–828.
- Egozcue, J.J., and V. Pawlowsky-Glahn. 2011. Basic concepts and procedures. In Compositional data analysis: Theory and applications, ed. V. Pawlowsky-Glahn and A. Buccianti. Chichester, UK: Wiley.
- Egozcue, J.J., V. Pawlowsky-Glahn, et al. 2003. Isometric logratio transformations for compositional data analysis. Mathematical Geology 35 (3): 279–300.
- Eisen, M.B., P.T. Spellman, et al. 1998. Cluster analysis and display of genome-wide expression patterns. Proceedings of the National Academy of Sciences of the United States of America 95 (25): 14863–14868.
- Erb, I., and C. Notredame. 2016. How should we measure proportionality on relative gene expression data? Theory in Biosciences 135: 21–36.
- Erb, I., T. Quinn, et al. 2017. Differential proportionality—A normalization-free approach to differential gene expression. bioRxiv.
- Fernandes, A.D., J.M. Macklaim, et al. 2013. ANOVA-like differential expression (ALDEx) analysis for mixed population RNA-seq. PLoS ONE 8 (7): e67019.
- Fernandes, A.D., J.N.S. Reid, et al. 2014. Unifying the analysis of high-throughput sequencing datasets: Characterizing RNA-seq, 16S rRNA gene sequencing and selective growth experiments by compositional data analysis. Microbiome 2 (1): 15.
- Filzmoser, P., K. Hron, et al. 2012. Interpretation of multivariate outliers for compositional data. Computers & Geosciences 39: 77–85.
- Finucane, M.M., T.J. Sharpton, et al. 2014. A taxonomic signature of obesity in the microbiome?
- Getting to the guts of the matter. PLoS ONE 9 (1): e84689.
- Friedman, J., and E.J. Alm. 2012a. Inferring correlation networks from genomic survey data. PLoS Computational Biology 8 (9): 20.
- Friedman, J., and E.J. Alm. 2012b. Inferring correlation networks from genomic survey data. PLoS Computational Biology 8 (9): e1002687.
- Gabriel, K.R. 1971. The biplot graphic display of matrices with application to principal component analysis. Biometrika 58 (3): 453–467.
- Gabriel, K.R. 1981. Biplot display of multivariate matrices for inspection of data and diagnosis. In Interpreting multivariate data, ed. V. Barnett. London: Wiley.
- Gabriel, K.R., and C.L. Odoroff. 1990. Biplots in biomedical research. Statistics in Medicine 9 (5): 469–485.
- Galton, F. 1879. The geometric mean, in vital and social statistics. Proceedings of the Royal Society of London 29: 365–366.
- Gevers, D., S. Kugathasan, et al. 2014. The treatment-naïve microbiome in new-onset Crohn’s disease. Cell Host & Microbe 15 (3): 382–392.
- Gloor, G.B., and G. Reid. 2016. Compositional analysis: A valid approach to analyze microbiome high-throughput sequencing data. Canadian Journal of Microbiology 62 (8): 692–703.
- Gloor, G.B., J.R. Wu, et al. 2016. It’s all relative: Analyzing microbiome data as compositions. Annals of Epidemiology 26 (5): 322–329.
- Gotelli, N.J. 2008. A primer of ecology. Sunderland, MA: Sinauer Associates Inc.
- Greenacre, M.J. 1993. Biplots in correspondence analysis. Journal of Applied Statistics 20 (2): 251–269.
- Hron, K., M. Templ, et al. 2010. Exploratory compositional data analysis using the R-package robCompositions. In Proceedings 9th International Conference on Computer Data Analysis and Modeling, vol. 1, ed. S. Aivazian, P. Filzmoser, and Y. Kharin, 179–186. Minsk: Belarusian State University.
- Knights, D., E.K. Costello, et al. 2011. Supervised classiﬁcation of human microbiota. FEMS Microbiology Reviews 35 (2): 343–359.
- Kroonenberg, P.M. 2008. Applied multiway data analysis. Hoboken, NJ: Wiley.
- Kurtz, Z.D., C.L. Müller, et al. 2015. Sparse and compositionally robust inference of microbial ecological networks. PLoS Computational Biology 11 (5): e1004226.
- Law, C.W., Y. Chen, et al. 2014. Voom: Precision weights unlock linear model analysis tools for RNA-seq read counts. Genome Biology 15 (2): R29.
- Ley, R.E., F. Bäckhed, et al. 2005. Obesity alters gut microbial ecology. Proceedings of the National Academy of Sciences of the United States of America 102 (31): 11070–11075.
- Ley, R.E., P.J. Turnbaugh, et al. 2006. Microbial ecology: Human gut microbes associated with obesity. Nature 444 (7122): 1022–1023.
- Li, H. 2015. Microbiome, metagenomics, and high-dimensional compositional data analysis. Annual Review of Statistics and Its Application 2: 73–94.
- López-Kleine, L., L. Leal, et al. 2013. Biostatistical approaches for the reconstruction of gene co-expression networks based on transcriptomic data. Brieﬁngs in Functional Genomics 12 (5): 457–467.
- Lovell, D., W. Müller, et al. 2011. Proportions, percentages, PPM: Do the molecular biosciences treat compositional data right? In Compositional Data Anal: Theory and Applications, ed.
- V. Pawlowsky-Glahn and A. Buccianti, 191–207. Chichester, UK: Wiley.
- Lovell, D., V. Pawlowsky-Glahn, et al. 2015. Proportionality: A valid alternative to correlation for relative data. PLoS Computational Biology 11 (3): e1004075.
- Mandal, S., W. Van Treuren, et al. 2015. Analysis of composition of microbiomes: A novel method for studying microbial composition. Microbial Ecology in Health and Disease 26: 27663.
- Martin Bland, J., and D. Altman. 1986. Statistical methods for assessing agreement between two methods of clinical measurement. The Lancet 327 (8476): 307–310.
- Martín-Fernández, J.A., C. Barceló-Vidal, et al. 2003. Dealing with zeros and missing values in compositional data sets using nonparametric imputation. Mathematical Geology 35 (3): 253–278.
- Martín-Fernández, J.A., J. Palarea-Albaladejo, et al. 2011. Dealing with zeros. In Compositional data analysis, ed. V. Pawlowsky-Glahn and A. Buccianti, 43–58. Chichester, UK: Wiley.
- Martín-Fernández, J.A., K. Hron, et al. 2012. Model-based replacement of rounded zeros in compositional data: Classical and robust approaches. Computational Statistics & Data Analysis 56: 2688–2704.
- Martín-Fernández, J.-A., K. Hron, et al. 2015. Bayesian-multiplicative treatment of count zeros in compositional data sets. Statistical Modelling 15 (2): 134–158.
- Mateu-Figueras, G., and V. Pawlowsky-Glahn. 2008. A critical approach to probability laws in geochemistry. Mathematical Geosciences 40 (5): 489–502.
- Mateu-Figueras, G., V. Pawlowsky-Glahn, et al. 2011. The principle of working on coordinates. In Compositional data analysis: Theory and applications, ed. V. Pawlowsky-Glahn and A. Buccianti. Chichester, UK: Wiley.
- McAlister, D. 1879. The law of the geometric mean. Proceedings of the Royal Society of London 29: 367–376.
- McCullagh, P., and J.A. Nelder. 1989. Generalized linear models. Boca Raton, FL: Chapman and Hall/CRC.
- Obayashi, T., and K. Kinoshita. 2011. COXPRESdb: A database to compare gene coexpression in seven model animals. Nucleic Acids Research 39 (Suppl 1): D1016–D1022.
- Palarea-Albaladejo, J., and J.A. Martín-Fernández. 2008. A modiﬁed EM alr-algorithm for replacing rounded zeros in compositional data sets. Computers & Geosciences 34 (8): 902–917.
- Palarea-Albaladejo, J., and J.A. Martín-Fernández. 2015. zCompositionsd—R package for multivariate imputation of left-censored data under a compositional approach. Chemometrics and Intelligent Laboratory Systems 143: 85–96.
- Palarea-Albaladejo, J., J.A. Martín-Fernández, et al. 2007. A parametric approach for dealing with compositional rounded zeros. Mathematical Geology 39 (7): 625–645.
- Pawlowsky-Glahn, V., and A. Buccianti. 2011. Compositional data analysis: Theory and applications. Chichester, UK: Wiley.
- Pawlowsky-Glahn, V., J.J. Egozcue, et al. 2015. Modeling and analysis of compositional data. UK: Wiley.
- Pearson, K. 1897. Mathematical contributions to the theory of evolution. On a form of spurious correlation which may arise when indices are used in the measurement of organs. Proceedings of the Royal Society of London LX: 489–502.
- Quinn, T., M.F. Richardson, et al. 2017. Propr: An R-package for identifying proportionally abundant features using compositional data analysis. bioRxiv.
- Shu, M., Y. Wang, et al. 2013. Fermentation of propionibacterium acnes, a commensal bacterium in the human skin microbiome, as skin probiotics against methicillin-resistant staphylococcus aureus. PLoS ONE 8 (2): e55380.
- Smyth, G.K. 2004. Linear models and empirical Bayes methods for assessing differential expression in microarray experiments. Statistical Applications in Genetics and Molecular Biology 3 (1): 1–25.
- Smyth, G.K. 2005. Limma: Linear models for microarray data. In Bioinformatics and computational biology solutions using R and bioconductor, ed. R. Gentleman, V. Carey, S. Dudoit, and W.H.R. Irizarry. New York: Springer.
- Sweeney, T.E., and J.M. Morton. 2013. The human gut microbiome: A review of the effect of obesity and surgically induced weight loss. JAMA Surgery 148 (6): 563–569.
- Sze, M.A., and P.D. Schloss. 2016. Looking for a signal in the noise: Revisiting obesity and the microbiome. MBio 7 (4): e01018–e01016.
- Templ, M., K. Hron, et al. 2011. RobCompositions: An R-package for robust statistical analysis of compositional data. In Compositional data analysis: Theory and applications, ed.
- V. Pawlowsky-Glahn and A. Buccianti, 341–355. Chichester, UK: Wiley. https://doi.org/10. 1002/9781119976462.ch25.
- The Human Microbiome Project Consortium. 2012. Structure, function and diversity of the healthy human microbiome. Nature 486 (7402): 207–214.
- Tsilimigras, M.C.B., and A.A. Fodor. 2016. Compositional data analysis of the microbiome: Fundamentals, tools, and challenges. Annals of Epidemiology 26 (5): 330–335.
- Turnbaugh, P.J., R.E. Ley, et al. 2006. An obesity-associated gut microbiome with increased capacity for energy harvest. Nature 444 (7122): 1027–1031.
- Turnbaugh, P.J., M. Hamady, et al. 2009. A core gut microbiome in obese and lean twins. Nature 457 (7228): 480–484.
- van den Boogaart, K.G., and R. Tolosana-Delgado. 2008. “Compositions”: A uniﬁed R package to analyze compositional data. Computers & Geosciences 34 (4): 320–338.
- van den Boogaart, K.G., and R. Tolosana-Delgado. 2013. Analyzing compositional data with R. Berlin, Heidelberg: Springer.
- van den Boogaart, K.G., R. Tolosana, et al. 2014. “Compositions: Compositional Data Analysis.
- R package version 1.40–1. http://CRAN.R-project.org/package=compositions.
- Walters, W.A., Z. Xu, et al. 2014. Meta-analyses of human gut microbes associated with obesity and IBD. FEBS Letters 588 (22): 4223–4233.
- Weiss, S.J., Z. Xu, et al. 2015. Effects of library size variance, sparsity, and compositionality on the analysis of microbiome data. PeerJ Preprints 3: e1157v1151.
- Zhang, B., and S. Horvath. 2005. A general framework for weighted gene co-expression network analysis. Statistical Applications in Genetics and Molecular Biology 4: 12.
- Zuur, A.F., E.N. Ieno, et al. 2009. Mixed effects models and extensions in ecology with R. New York, NY: Springer Science & Business Media, LLC.

